src/settings.rs

```diff
@@ -169,7 +169,7 @@ impl Default for Settings {
             exclude_dma: Some("[798, 583, 740]".to_owned()),
             connect_timeout: 2,
             request_timeout: 5,
-            excluded_countries_200: true,
+            excluded_countries_200: false,
```

Comment:
It sounds like this setting was introduced to work around a bug in older versions of Firefox. Shall we get a follow-up on file to remove the setting entirely? 😄

Reply:
Good point, let me file a ticket for that. We will deploy this change to prod and see how Firefox behaves. If nothing goes south, we can drop this treatment in contile.

Reply:
Here is the ticket: https://mozilla-hub.atlassian.net/browse/DISCO-2518.